From a52c525710a504634502424e182a6fbaa68011d7 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Mon, 22 May 2023 14:27:06 +0800
Subject: [PATCH 4/6] amlbian: increase ion-fb buffer size [1/1]

PD#SWPL-123331

Problem:
Stuck when switching users

Solution:
increase ion-fb buffer size.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: I2af178100f125ae532b1748679b021d0d5ae98ed
---
 arch/arm64/boot/dts/amlogic/g12b_a311d_w400_debian.dts  | 2 +-
 arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_debian.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_debian.dts b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_debian.dts
index 2eefbb1b..19d19f41 100644
--- a/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_debian.dts
+++ b/arch/arm64/boot/dts/amlogic/g12b_a311d_w400_debian.dts
@@ -84,7 +84,7 @@
 		ion_fb_reserved:linux,ion-fb {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x0 0x0>;
+			size = <0x0 0x20000000>;
 			alignment = <0x0 0x400000>;
 		};
 		dmaheap_cma_reserved:heap-gfx {
diff --git a/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_debian.dts b/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_debian.dts
index 94236a59..f6010a70 100644
--- a/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_debian.dts
+++ b/arch/arm64/boot/dts/amlogic/sm1_s905d3_ac200_debian.dts
@@ -84,7 +84,7 @@
 		ion_fb_reserved:linux,ion-fb {
 			compatible = "shared-dma-pool";
 			reusable;
-			size = <0x0 0x0>;
+			size = <0x0 0x20000000>;
 			alignment = <0x0 0x400000>;
 		};
 		dmaheap_cma_reserved:heap-gfx {
-- 
2.25.1

